docs/parameters.yaml

```
@@ -2677,7 +2677,11 @@ components: ["origin", "cache", "director", "registry"]
 ############################
 name: Shoveler.Enable
 description: |+
-  Enable XRootD monitoring shoveler: https://github.com/opensciencegrid/xrootd-monitoring-shoveler.
+  The shoveler gathers UDP monitoring messages from XRootD servers and sends them to a message bus, such
```

```suggestion
  Enable the XRootD monitoring shoveler. The shoveler gathers UDP monitoring messages from XRootD servers and sends them to a message bus, such
```

I think it's still important to explicitly state what this knob does